snap/snapcraft.yaml

```diff
@@ -13,6 +13,7 @@ confinement: classic
 apps:
   snapcraft:
     command: bin/snapcraft
+    completer: snapcraft-completion
```

Comment:
This is also not related to this pr.

Reply:
It is because I rebased, but this PR is not against master, and even though the target branch is now updated, github missed out on that fact.